wagtail/embeds/models.py

```diff
@@ -22,20 +25,19 @@ class Embed(models.Model):
     If an instance of this model is deleted, it will be automatically refetched
     next time the embed code is needed.
     """
-    url = models.URLField()
+    url = URLField()
```

Comment:
As mentioned previously, we might as well use `TextField` here and drop the custom field/related tests